src/llv8.cc

```diff
@@ -1838,12 +1847,16 @@ v8::Value JSObject::GetArrayElement(int64_t pos, Error& err) {
 }
 
 std::string JSArray::Inspect(InspectOptions* options, Error& err) {
-  Smi length = Length(err);
+  int64_t length = GetArrayLength(err);
   if (err.Fail()) return std::string();
 
-  std::string res = "<Array: length=" + length.ToString(err);
+  std::string res = "<Array: length=" + std::to_string(length);
   if (options->detailed) {
-    std::string elems = InspectElements(err);
+    int64_t display_length = length;
+    if (length > options->array_length) {
+      display_length = options->array_length;
+    }
```

Comment:
`int64_t display_length = std::min(length, options->array_length);`?  Arguably a little easier to read.

If not that, can you `s/length/display_length/` in the comparison?